test/Microsoft.AspNet.Mvc.ModelBinding.Test/ValueProviders/FormValueProviderFactoryTests.cs

```diff
@@ -32,7 +33,9 @@ public void GetValueProvider_ReturnsNull_WhenContentTypeIsNotFormUrlEncoded()
         [Theory]
         [InlineData("application/x-www-form-urlencoded")]
         [InlineData("application/x-www-form-urlencoded;charset=utf-8")]
-        public void GetValueProvider_ReturnsValueProviderInstaceWithInvariantCulture(string contentType)
+        [InlineData("multipart/form-data")]
+        [InlineData("multipart/form-data;charset=utf-8")]
```

Comment:
I think you might need a space after the semi-colon here. Check @Tratcher's pr changes for strongly typed header.